src/parsec_service/install_parsec_linux.md

```diff
@@ -15,48 +15,62 @@ can be a space or comma-separated subset of: `mbed-crypto-provider`, `pkcs11-pro
 `tpm-provider`. Choose the providers you want to install depending on what is available on the
 platform.
 
-Create and log in to a new user named `parsec`.
+Create the Parsec socket directory.
 
 ```
-sudo useradd -m parsec
-sudo passwd parsec
-su --login parsec
+mkdir /tmp/parsec
 ```
 
-In its home directory, pull and install Parsec as a daemon.
+In a deployment **without an Identity Provider**, create the `parsec-clients` group and set the
+correct permissions on the socket folder. Mutually trusted Parsec Clients will need to be in that
+group.
 
 ```
-git clone https://github.com/parallaxsecond/parsec.git
-cargo install --features $DESIRED_FEATURES --path parsec
+sudo groupadd parsec-clients
+sudo chown :parsec-clients /tmp/parsec
+sudo chmod 750 /tmp/parsec
 ```
 
-Copy and adapt the [configuration](configuration.md) you want to use. For a secure deployment, make
-sure to activate the `log_error_details` option and to use a `trace` log level.
+For example, adding the imaginary `parsec-client-1` user to the `parsec-clients` group:
 
 ```
-cp parsec/config.toml config.toml
+sudo usermod -a -G parsec-clients parsec-client-1
 ```
 
-Create the Parsec socket directory.
+Users just added to that group might need to log-out and log-in again to make sure the change apply.
+
+In a deployment **with an Identity Provider**, set the correct permissions on the socket folder.
 
 ```
-mkdir /tmp/parsec
+sudo chmod 755 /tmp/parsec
 ```
 
-In a deployment **without an Identity Provider**, create the `parsec-clients` group and set the
-correct permissions on the socket folder. Mutually trusted Parsec Clients will need to be in that
-group.
+Create and log in to a new user named `parsec`.
 
 ```
-sudo groupadd parsec-clients
-sudo chown :parsec-clients /tmp/parsec
-sudo chmod 750 /tmp/parsec
+sudo useradd -m parsec
+sudo passwd parsec
+su --login parsec
 ```
 
-In a deployment **with an Identity Provider**, set the correct permissions on the socket folder.
+Depending on which features of Parsec the `parsec` user is going to use, it might need to be given
+more privileges in order to access some resources on the system. Refer to the
+[Providers](providers.md) page for more information.
+
+In its home directory, pull and install Parsec as a daemon. If a Rust toolchain is not available
+widely on the system, it will need to be [installed](https://www.rust-lang.org/tools/install) for
+that specific user.
 
 ```
-sudo chmod 755 /tmp/parsec
+git clone https://github.com/parallaxsecond/parsec.git
+cargo install --features $DESIRED_FEATURES --path parsec
+```
+
+Copy and adapt the [configuration](configuration.md) you want to use. For a secure deployment, make
+sure to activate the `log_error_details` option and to use a `trace` log level.
```

Comment:
It's probably not a good idea to change this right now, but I still don't think either `log_error_details` or `trace` help with security. The log flag would print more details of errors - potentially helpful if a problem occurs, but doesn't add anything to security (actually, as written in the config example file, it could lead to some data leakage). `trace` is a (very) weak protection against repudiation - while it might have some value, the sheer amount of data it would is probably not warranted in a production environment.

Again, not for now, we can refactor at some later point.

Reply:
Agree that logging might not be the best mitigation against repudiation and would be ok to remove it if we find another mitigation to the related threats.

Reply:
Unfortunately I don't think there's much we can do without mutual authentication 😞